README.md

```diff
@@ -30,3 +30,10 @@ $ cmake .. -DCMAKE_INSTALL_PREFIX=~/vmemcache-bin
 $ make
 $ make install
 ```
+
+# Statistics #
+
+Statistics are enabled by default. They can be disabled at the compile time
```

Comment:
Do we really want to maintain both? Did we measure the impact of stats?

Reply:
What do you mean by 'both'? Yes, it is a bit faster

Reply:
Holy crap, on a 96-core, 2-socket machine: bench_simul type=index: with stats, 6.8 µs, without stats 1.3 µs!!!!!1!!1111!-e^(iπ)!!

Me wants.

Looks like we should look hard into decreasing the impact of gathering stats if we're to leave them on by default.

Reply:
@ldorau by "both" I meant "both options".
@kilobyte how the numbers look like when you limit the benchmark to one numa node?

Reply:
Both - yes

Reply:
bench_simul n_threads=48 type=index

||both sockets|numactl -N 0|
|-|-|-|
|STATS|5.3|2.6|
|NO_STATS|0.8|0.52|

Reply:
interesting, it seems we need per-thread stats...